### ISSUE TITLE
Star Ocean(TM) - The Last Hope(TM) - 4k & FULL HD Remaster (609150)

### ISSUE THREAD

**Issue by Glog78** [ValveSoftware/Proton#1268]
Systeminfo -> http://termbin.com/rdxl
Game crashes on start / most likely denuvo if not an additional layer of protection.
Log -> http://termbin.com/pmvg

**Comment by Glog78** [comment]
http://wine.1045685.n8.nabble.com/Bug-45768-New-PES-2019-demo-launchs-correctly-fails-upon-connecting-server-GnuTLS-does-not-support-a-td5974345.html << maybe this helps cause there are for sure the mentioned messages inside of the log (with 3.16-2):

90150.009:0024:0025:trace:module:load_dll looking for L"crypt32.dll" in L"Z:\\mnt\\opt1\\steamdirs\\******\\SteamLibrary_new\\steamapps\\common\\STAR OCEAN - THE LAST HOPE - 4K & Full HD Remaster;C:\\Program Files (x86)\\Steam;C:\\windows\\system32;C:\\windows\\system;C:\\windows;.;C:\\windows\\system32;C:\\windows;C:\\windows\\system32\\wbem"
90150.009:0024:0025:trace:module:load_dll Found L"C:\\windows\\system32\\crypt32.dll" for L"crypt32.dll" at 0x7f60a7d70000, count=3
90150.021:0024:0025:fixme:bcrypt:key_asymmetric_verify GnuTLS does not support algorithm 11 with hash len 48
90150.021:0024:0025:fixme:crypt:CNG_VerifySignature Failed to verify signature: c0000002
90150.023:0024:0025:fixme:bcrypt:key_asymmetric_verify GnuTLS does not support algorithm 11 with hash len 48
90150.023:0024:0025:fixme:crypt:CNG_VerifySignature Failed to verify signature: c0000002
90150.028:0024:0025:fixme:bcrypt:key_asymmetric_verify GnuTLS does not support algorithm 11 with hash len 48
90150.028:0024:0025:fixme:crypt:CNG_VerifySignature Failed to verify signature: c0000002
90150.038:0024:0025:fixme:bcrypt:key_asymmetric_verify GnuTLS does not support algorithm 11 with hash len 32
90150.038:0024:0025:fixme:crypt:CNG_VerifySignature Failed to verify signature: c0000002
90150.039:0024:0025:fixme:bcrypt:key_asymmetric_verify GnuTLS does not support algorithm 11 with hash len 32
90150.039:0024:0025:fixme:crypt:CNG_VerifySignature Failed to verify signature: c0000002
90150.047:0024:0025:fixme:bcrypt:key_asymmetric_verify GnuTLS does not support algorithm 11 with hash len 48
90150.047:0024:0025:fixme:crypt:CNG_VerifySignature Failed to verify signature: c0000002

....

**Comment by kisak-valve** [comment]
Hello @Glog78, the wine patch should go to upstream wine if it has not been already.

**Comment by tom-sn** [comment]
Still no launch on latest 4.2-4 proton (Fedora 30), just show running status for a few second then back to nothing.
There are no more fixme in the log now.

[steam-609150.log](https://github.com/ValveSoftware/Proton/files/3183498/steam-609150.log)

**Comment by kisak-valve** [comment]
The store page indicates that this game uses Denuvo.

**Comment by CygnusTerminal** [comment]
Update as of July 18th, 2020. Game opens with Proton 5.0-9, but trying to change settings crashes the game to desktop.
[steam-609150.log](https://github.com/ValveSoftware/Proton/files/4941092/steam-609150.log)

**Comment by kisak-valve** [comment]
Compatibility Report - STAR OCEAN - THE LAST HOPE

Issue transferred from https://github.com/ValveSoftware/Proton/issues/4226.
@Akin2Silver posted on 2020-09-29T03:59:20:

---
name: Compatibility Report
about: Game compatibility issues.

---

# Compatibility Report
- Name of the game with compatibility issues: STAR OCEAN - THE LAST HOPE - 4K & Full HD Remaster 
- Steam AppID of the game: 609150

## System Information
- GPU: AMD Radeon 5700XT
- Driver/LLVM version: Mesa 20.1.7-1 / LLVM 10.0.1-2
- Kernel version: 5.8.6-1
- Link to full system information report as [Gist](https://gist.github.com/):[system info](https://gist.github.com/Akin2Silver/4cb336a355f4a3a2faee30ff8d340bb6)
- Proton version: 5.0-9, 4.11-13, 4.2-9

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.


[steam-609150.log](https://github.com/ValveSoftware/Proton/files/5296270/steam-609150.log)
## Symptoms 

Game crashes when selecting any option from the main menu. 
Proton log only shows process exceptions no extra info. 

## Reproduction

Launch game with basic proton support, select `new game` from main menu.

**Comment by Jhackler** [comment]
entering setting menu chrashes it but it gets to the start screen lol

**Comment by ghost** [comment]
This game apparently just works now. Wonder what changes helped it.

**Comment by Jhackler** [comment]
> This game apparently just works now. Wonder what changes helped it.

yeah I read that so tried proton experimental and it does indeed just work
